jax/random.py

```diff
@@ -12,6 +12,68 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""Utilities for pseudo-random number generation.
+
+The ``jax.random`` package provides a number of routines for deterministic
+generation of sequences of pseudorandom numbers.
+
+Basic usage
+-----------
+
+>>> rng = jax.random.PRNGKey(seed)
```

Comment:
Slight preference for calling this `key`, since `rng` implies it's more of a heavyweight object imo, but this is a minor preference so feel free to ignore it.

Reply:
Done